ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -265,16 +265,33 @@ pub(crate) struct Instruction {
     name: RefCell<Option<String>>,
 }
 
-impl Instruction {
+impl<'a> Instruction {
```

Comment:
Since `Instruction` doesn't have a lifetime, I think this lifetime should be moved directly into those functions that need it (e.g. `get_callee`).

Reply:
Oops, that's probably a left-over from an earlier design.

Reply:
Fixed in 02631b74d6a455fe7aaad5183daf16098dd5797e